src/Microsoft.Framework.Caching.SqlServer/SqlQueries.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Framework.Caching.SqlServer
+{
+    internal class SqlQueries
+    {
+        private const string CreateTableFormat = "CREATE TABLE [{0}](" +
+            "Id nvarchar(100)  NOT NULL PRIMARY KEY, " +
+            "Value varbinary(MAX) NOT NULL, " +
+            "ExpiresAtTimeUTC datetimeoffset NOT NULL, " +
+            "SlidingExpirationInTicks bigint NULL," +
+            "AbsoluteExpiration datetimeoffset NULL)";
+
+        private const string CreateNonClusteredIndexOnExpirationTimeFormat
+            = "CREATE NONCLUSTERED INDEX Index_ExpiresAtTimeUTC ON [{0}](ExpiresAtTimeUTC)";
+
+        // Used for getting table schema when trying to 'Connect' to the database
+        private const string GetTableSchemaFormat = "SELECT * FROM [{0}]";
+
+        private const string GetCacheItemFormat =
+            "SELECT Id, Value, ExpiresAtTimeUTC, SlidingExpirationInTicks, AbsoluteExpiration " +
+            "FROM [{0}] WHERE Id = @Id AND @UtcNow <= ExpiresAtTimeUTC";
+
+        private const string AddCacheItemFormat = "INSERT INTO [{0}] " +
+            "(Id, Value, ExpiresAtTimeUTC, SlidingExpirationInTicks, AbsoluteExpiration) " +
+            "VALUES (@Id, @Value, @ExpiresAtTimeUTC, @SlidingExpirationInTicks, @AbsoluteExpiration)";
+
+        private const string UpdateCacheItemFormat = "UPDATE [{0}] SET Value = @Value, " +
+            "ExpiresAtTimeUTC = @ExpiresAtTimeUTC  WHERE Id = @Id";
+
+        private const string DeleteCacheItemFormat = "DELETE FROM [{0}] WHERE Id = @Id";
+
+        private const string UpdateCacheItemExpirationFormat = "UPDATE [{0}] SET ExpiresAtTimeUTC = @ExpiresAtTimeUTC " +
+            "WHERE Id = @Id";
+
+        public const string DeleteExpiredCacheItemsFormat = "DELETE FROM [{0}] WHERE @UtcNow > ExpiresAtTimeUTC";
+
+        public SqlQueries(string tableName)
+        {
+            //TODO: error checks
+
+            CreateTable = string.Format(CreateTableFormat, tableName);
```

Comment:
Is it ok to assume that the user of the command line tool is trusted? I imagine in most situations it better be but otherwise we should probably escape (in the SQL sense) the tableName before we pass it to the `Format()` method. Otherwise it seems it would be pretty easy to get this tool to execute arbitrary SQL.